.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Report an issue or bug with the Quantum Core X.
+title: "bug: "
+labels: [ "type/bug" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to help us diagnose and fix the issue. Before submitting a bug report, please check the [Reporting Bugs](https://github.com/MeikelLP/quantum-core-x/blob/master/CONTRIBUTING.md#reporting-bugs) section in the contributing guidelines.
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: A clear and concise description of what actually happened.
+      placeholder: Describe the bug you encountered.
+    validations:
+      required: true
+
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: How to reproduce?
+      description: Steps to reproduce the behavior.
+      placeholder: Steps to reproduce the behavior.
+    validations:
+      required: true
+
+  - type: textarea
+    id: tracebacks
+    attributes:
+      label: Tracebacks
+      description: If applicable, paste the full traceback here.
```

Comment:
@MeikelLP are there debug or tracing instruction available? E.g. how to obtain a meaningful stack trace or log file, or maybe even how to run the application with `DEBUG` logging or tracing enabled for reproduction?

If so, we could add them here or link to them.

Reply:
Stacktraces are always logged if an exception is thrown. You should be able to change the log level when setting `"Serilog:MinimumLevel:Default": "Debug"`  in the appsettings

Reply:
I've added a note to reflect that, is that okay?

![image](https://github.com/user-attachments/assets/1111f716-0a37-4e23-a9cd-0c0cd9659427)

Reply:
If you're good with this as is in the screenshot then I think this PR is ready to merge @MeikelLP